### ISSUE TITLE
BUG - Masked Input

### ISSUE THREAD

**Issue by cacacobra** [plentz/jquery-maskmoney#45]
Ainda, to tendo problemas com os seguintes plugins:

jQuery            @Version: 1.7.2
MaskMoney   @Version: 1.4.1
MaskedInput @Version: 1.3

Basicamente, se o MaskMoney , for incluido depois do MaskedInput, o MaskedInput  não funciona.
Então para ter os dois funcionando, eu preciso chama-los assim:

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>

<script type="text/javascript" src="js/maskMoney.js"></script>

<script src="js/jquery.maskedinput.js" type="text/javascript"></script>

O contrário não da certo.

Vi em uma postagem anterior que esse problema teria sido corrigido, mas eu ainda estou visualizando o BUG na minha aplicação.
